root/statistics/CoverArt.js

```diff
@@ -0,0 +1,202 @@
+9/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l_statistics, ln_statistics} from '../static/scripts/common/i18n/statistics';
+import {withCatalystContext} from '../context';
+import manifest from '../static/manifest';
+
+import {formatCount, formatPercentage} from './utilities';
+import StatisticsLayout from './StatisticsLayout';
+import type {CoverArtStatsT} from './types';
+
+const nameOrNull = (name: string, defaultName: string) => {
+  if (name === 'null') {
+    return defaultName;
+  } else {
+    return l_statistics(name);
+  }
+};
+
+const CoverArt = ({
+  $c,
+  dateCollected,
+  releaseTypeStats,
+  releaseStatusStats,
+  releaseFormatStats,
+  stats,
+  typeStats,
+}: CoverArtStatsT) => {
+  const oneToTwentyNine = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29];
+  return (
+    <StatisticsLayout fullWidth page="coverart" title={l_statistics('Cover Art')}>
+      {manifest.css('statistics')}
+      <p>{l_statistics('Last updated: {date}',
+        {__react: true, date: dateCollected})}
+      </p>
+      <h2>{l_statistics('Basics')}</h2>
+      {/* TODO: check why this seems to just not exist rather than be 0 */}
```

Comment:
Do you remember what that's about? Maybe we can figure it out.

Reply:
IIRC stats.data['count.release.has_caa'] was just not returned, rather than returned as 0. But I just figured out why - that's marked PRIVATE for... some reason: https://github.com/metabrainz/musicbrainz-server/blob/2dc83f63299776995cdc4a924ea8b93ded5676a4/lib/MusicBrainz/Server/Data/Statistics.pm#L1215

Reply:
So it should work fine in prod, anyway, and it's just an issue in RT_STANDALONE, same as the info about editors.